### PR TITLE
Refine Microk8s and Multipass docs

### PR DESCRIPTION
**Pull request by RFMVasconcelos**
The intro on these docs was confusing and outdated regarding new Microk8s capabilities.

### CODE REVIEW

**Comment by joeliedtke** [comment]
Preview: https://deploy-preview-2019--competent-brattain-de2d6d.netlify.app/docs/started/workstation/getting-started-multipass/

File: content/en/docs/started/workstation/getting-started-multipass.md [hunk 1]
```diff
@@ -7,21 +7,15 @@ weight = 60
 {{% alpha-status 
   feedbacklink="https://github.com/kubeflow/kubeflow/issues" %}}
 
-This document outlines the steps that you can take to get your local installation of Kubeflow running on top of Microk8s, a small enterprise Kubernetes cluster. Microk8s requires Linux; if you are not on a Linux system, you can use Multipass to create a Linux VM (virtual machine) on your native hypervisor.
 
 ## Introduction
 
-If you already have Ubuntu or Linux that supports [snaps](https://snapcraft.io/), you can easily install Kubernetes using [Microk8s](https://microk8s.io/). You can jump to the **Install Kubeflow using Microk8s** section below.
+This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, MacOs and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website.
```
Comment (joeliedtke) [inline content/en/docs/started/workstation/getting-started-multipass.md:13]: MacOs and -> macOS, and

File: content/en/docs/started/workstation/getting-started-multipass.md [hunk 2]
```diff
@@ -7,21 +7,15 @@ weight = 60
...
 
 ## Introduction
 
-If you already have Ubuntu or Linux that supports [snaps](https://snapcraft.io/), you can easily install Kubernetes using [Microk8s](https://microk8s.io/). You can jump to the **Install Kubeflow using Microk8s** section below.
+This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, MacOs and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
 
-If you don't have a Linux system already, or you would like to confine your Kubeflow to a disposable machine, then [Create a VM with Multipass](https://multipass.run/) first and then follow the instructions below. That will get you an Ubuntu machine that can be used to install Kubernetes and Kubeflow.
+Alternatively, you can install Microk8s on a Linux appliance using [Multipass](https://multipass.run/), if you would like to confine your Kubeflow to a disposable Linux virtual machine on your Windows or MacOS host.
```
Comment (joeliedtke) [inline content/en/docs/started/workstation/getting-started-multipass.md:15]: Suggested revision:

Alternatively, to deploy Kubeflow within a displosable Linux virtual machine, you can install Microk8s on a Linux appliance using [Multipass](https://multipass.run/) on Windows or MacOS.

File: content/en/docs/started/workstation/getting-started-multipass.md (hunk 1 above)
Comment (RFMVasconcelos) [reply content/en/docs/started/workstation/getting-started-multipass.md:13]: Nice catch!

Comment (8bitmp3) [inline content/en/docs/started/workstation/getting-started-multipass.md]: I propose to simplify your first sentence by cutting some words out, while retaining the meaning. At the moment, it's a bit verbose: "This doc outlines the steps that you can take to..."

```diff
This 
- document outlines the steps that you can take 
+ guide will show you how
- to get your local 
+ to deploy Kubeflow
- installation of Kubeflow 
- running on top of 
+ on [Microk8s](https://microk8s.io/)
- , a small enterprise Kubernetes cluster.
+ —a small enterprise Kubernetes cluster.
```

Comment (RFMVasconcelos) [reply content/en/docs/started/workstation/getting-started-multipass.md]: Awesome! Thank you for this review 🚀

**Comment by RFMVasconcelos** [comment]
@8bitmp3 could I ask you for a final review on this? I have implemented your suggestions in the last commit. 

I will be adding a few more PRs today to get these microk8s docs on point 🚀

File: content/en/docs/started/workstation/getting-started-multipass.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ weight = 60
 
 ## Introduction
 
-This document outlines the steps that you can take to get your local installation of Kubeflow running on top of [Microk8s](https://microk8s.io/), a small enterprise Kubernetes cluster. Microk8s is now available on Windows, macOS and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website. 
+This guide will show you how to deploy Kubeflow using [Microk8s](https://microk8s.io/) - a small enterprise Kubernetes cluster. Microk8s is now available on Windows, macOS and any Linux distribution that supports `snaps`. You can download it on the [Microk8s](https://microk8s.io/) website.
```
Comment (8bitmp3) [inline content/en/docs/started/workstation/getting-started-multipass.md:13]: After some studying, I realized that using a present tense is the commonly used practice.

```diff
- This guide will show you how to to deploy Kubeflow using [Microk8s]
+ This guide describes how to deploy Kubeflow using [Microk8s]
```

What do you think?

Comment (RFMVasconcelos) [reply content/en/docs/started/workstation/getting-started-multipass.md:13]: LGTM

Comment (RFMVasconcelos) [reply content/en/docs/started/workstation/getting-started-multipass.md]: Thank you!

**Comment by RFMVasconcelos** [comment]
Ended up doing all the changes on this PR for simplicity. I believe it is ready to merge.

**Comment by 8bitmp3** [comment]
@RFMVasconcelos I'll create a PR for your repo and explain the changes in detail here, so we can finalize this and get closer to the merge 👍

**Comment by RFMVasconcelos** [comment]
Hi @8bitmp3, any update on this front? 🚀 

Which changed do you think are still needed?

**Comment by RFMVasconcelos** [comment]
@joeliedtke, any chance you could review + approve this?

**Comment by 8bitmp3** [comment]
@RFMVasconcelos I split the installation and enabling steps in digestible chunks. Introduced other minor changes. LMKWYT! https://github.com/RFMVasconcelos/website/pull/1 (By the way, I noticed it's "MicroK8s" instead of "Microk8s" - when changing the format automatically, it also affected the command lines (`MicroK8s...`) - can you fix them if they affect the output).

**Comment by RFMVasconcelos** [comment]
@8bitmp3 Thank you for the help here! 

It seems that you started editing from an outdated commit, not this branch, so you reverted a few changes. Let me cherrypick from your changes the ones that are new and commit them directly here. sounds good?

Cheers!

**Comment by 8bitmp3** [comment]
Absolutely @RFMVasconcelos The dated branches really showed 🙂 👍

**Comment by 8bitmp3** [comment]
@RFMVasconcelos As discussed a few hours ago, since you don't have enough time today - see PR: https://github.com/RFMVasconcelos/website/pull/2 for https://github.com/kubeflow/website/pull/2019.

I studied the MicroK8s and Multipass docs again to make sure we don’t miss anything here. Some of the significant changes proposed:

- Change: “Microk8s” -> “MicroK8s”
- Add: Multipass gives an Ubuntu command line - and you can install MicroK8s on top of it
    - Add Multipass docs link, since the installation process for Muiltipass + MicroK8s is lengthier than for native MicroK8s
- Add :Multipass also supports Linux
- More sequential steps in Accessing the dashboard section
- Clarify: replace IP placeholders with own IP addresses (e.g. `<multipass_public_ip>`)
- Bolden: the UI elements in Linux
- Change: “Next steps” -> “Additional guides”

```diff
- description = “Quickly get Kubeflow running locally on built-in hypervisors with MicroK8s"
+ description = "Run Kubeflow locally on built-in hypervisors with MicroK8s" weight = 60
…
This guide describes how to deploy Kubeflow using Microk8s - a small enterprise Kubernetes cluster
+ - locally on built-in hypervisors with MicroK8s.
```

**Comment by RFMVasconcelos** [comment]
@8bitmp3 I made a few final changes after your merge.

**Comment by RFMVasconcelos** [comment]
@8bitmp3 I fixed text wrapping and also removed the alpha status, as this applies to applications, according to [version policy](https://www.kubeflow.org/docs/reference/version-policy/), not deployment methods.

**Comment by RFMVasconcelos** [comment]
WDYT?

File: content/en/docs/started/workstation/getting-started-multipass.md [hunk 4]
```diff
@@ -42,55 +33,48 @@ it, and then set up Kubeflow.
...
-   MicroK8s deployment. To do that, run the following command:
+3. Having installed MicroK8s, you can now enable common services on your MicroK8s deployment. To do that, run the following command:
 
     ```
     microk8s.enable dns dashboard storage
     ```
 
-    **Optional:** To enable NVIDIA GPU hardware support, also run
-    `MicroK8s.enable gpu`.
+    **Optional:** To enable NVIDIA GPU hardware support, also run `MicroK8s.enable gpu`.
```
Comment (8bitmp3) [inline content/en/docs/started/workstation/getting-started-multipass.md:41]: @RFMVasconcelos Can you please check if it's `microK8s.enable gpu` (lower caps)? if this works too, no worries

File: content/en/docs/started/workstation/getting-started-multipass.md [hunk 5]
```diff
@@ -42,55 +33,48 @@ it, and then set up Kubeflow.
...
-    Run the following command, where `<machine_public_ip>` is your machine's
-    public IP:
+    In the VM case, run the following command, where `<machine_public_ip>` is your machine's public IP:
 
     ```
     ssh -D9999 ubuntu@<machine_public_ip>
     ```
 
-    Or, you can check for the Multipass IP first with:
+    On Windows, macOS or multipass, you can check for the IP first with:
```
[the remark this answers is not in the document]
Comment (8bitmp3) [reply content/en/docs/started/workstation/getting-started-multipass.md:76]: 🙂 @RFMVasconcelos 

image

File: content/en/docs/started/workstation/getting-started-multipass.md (hunk 4 above)
Comment (RFMVasconcelos) [reply content/en/docs/started/workstation/getting-started-multipass.md:41]: Oh yeah, lower is better!

File: content/en/docs/started/workstation/getting-started-multipass.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (RFMVasconcelos) [reply content/en/docs/started/workstation/getting-started-multipass.md:76]: Hahah fixing!

**Comment by RFMVasconcelos** [comment]
Hey @8bitmp3 @joeliedtke!

It seems we finally got to a good point in this doc. 🚀  

Could I gently ask you to review/approve this PR?

File: content/en/docs/started/workstation/getting-started-multipass.md [hunk 6]
```diff
@@ -1,97 +1,99 @@
...
     ssh -D9999 multipass@<multipass_public_ip>
     ```
 
-* On your computer, go to `Settings > Network > Network Proxy`, and enable SOCKS proxy pointing to: `127.0.0.1:9999`.
+3. In your Linux operating system, go to **Settings** > **Network** > **Network Proxy**, and enable SOCKS proxy pointing to: `127.0.0.1:9999`.
 
-* Finally, (1) open a new web browser window and (2) access the link provided after you enable Kubeflow, e.g. `10.64.140.43.xip.io` (see previous step).
+4. Finally, access the Kubeflow dashboard by:
+    1. Opening a new web browser window.
+    2. Access the link provided after you have enabled Kubeflow (for example, `10.64.140.43.xip.io`).
```
Comment (8bitmp3) [inline content/en/docs/started/workstation/getting-started-multipass.md:91]: Maybe this should be "Accessing". Other than that, lgtm

Comment (RFMVasconcelos) [reply content/en/docs/started/workstation/getting-started-multipass.md]: Nice catch! Solved! 🚀

**Comment by RFMVasconcelos** [comment]
@joeliedtke @8bitmp3 I am not able to approve this, as it's my PR, could you please approve this?

**Pull request merged**